pages/blog/blog-post.js

```diff
@@ -0,0 +1,301 @@
+
+  &:hover {
+    color: #444444 !important;
+    opacity: 1;
+  }
+`
+
+const HorizontalRule = styled.hr`
+  border-width: 2px;
+  border-color: ${props => props.theme.pineCone};
+  margin-top: 4rem;
+`
+
+const ButtonContainer = styled.div`
+ display: flex;
+ flex-direction: row;
+ justify-content: space-between;
+`
+
+const BlogPostContent = (props) => {
+  const {
+    title,
+    metaDescription,
+    date,
+    heroImage,
+    content,
+    authorTeamMember,
+    authorExternal,
+    url,
+  } = props
+
+  const author = authorTeamMember ?
+    (<TeamMemberAuthor
+      imageUrl={authorTeamMember.image.url}
+      title={authorTeamMember.name}
+      subtitle={authorTeamMember.responsibilityArea}
+      email={authorTeamMember.email}
+    />) :
+    <p>{authorExternal}</p>
+
+  const pageUrl = `https://www.masifunde.de/${url.asPath}`
+  const shareMessage = ''
+
+  return (
+    <div>
+      <Head title={title} description={metaDescription} />
+
+      {heroImage && <Hero
+        heroSize="small"
+        backgroundPositionX="70%"
+        imageUrl={heroImage.url}
+      />}
+
+      <PageSection>
+        <div className="offset-lg-2 col-lg-8">
+          <DateContainer>
+            <h5>{date}</h5>
+          </DateContainer>
+          <BlogTitle>{title}</BlogTitle>
+          <BlogMarkdown source={content} />
+          <div className="row">
+            <AuthorContainer className="col-6">
+              <h4>{props.authorText}</h4>
+              {author}
+            </AuthorContainer>
+            <ShareContainer className="col-6">
+              <h4>{props.shareText}</h4>
+
+              <ShareButtonRow>
+                <FacebookShareButton
+                  url={pageUrl}
+                  quote={shareMessage}
+                >
+                  <SocialLink>
+                    <FaFacebook size={24} />
+                  </SocialLink>
+                </FacebookShareButton>
+
+                <TwitterShareButton
+                  url={pageUrl}
+                  title={shareMessage}
+                >
+                  <SocialLink>
+                    <FaTwitter size={24} />
+                  </SocialLink>
+                </TwitterShareButton>
+              </ShareButtonRow>
+
+            </ShareContainer>
+          </div>
+
+        </div>
+      </PageSection>
+
+    </div>
+  )
+}
+
+BlogPostContent.propTypes = {
+  title: PropTypes.string,
+  slug: PropTypes.string,
+  metaDescription: PropTypes.string,
+  date: PropTypes.string,
+  heroImage: PropTypes.shape(imageShape),
+  content: PropTypes.string,
+  authorTeamMember: PropTypes.shape(teamMemberShape),
+  authorExternal: PropTypes.string,
+}
+
+BlogPostContent.defaultProps = {
+  title: '',
+  slug: '',
+  metaDescription: '',
+  date: '',
+  heroImage: null,
+  content: '',
+  authorTeamMember: null,
+  authorExternal: '',
+}
+
+const BlogPost = props => (
+  props.error ?
+    BlogPostError(props) :
+    (
+      <div>
+        {BlogPostContent(props)}
```

Comment:
This is just another normal React component. Don't call it as a function, rather use regular jsx.
`<BlogPostContent {...props} />`